packages/babel-helper-create-class-features-plugin/src/fields.js

```diff
@@ -615,14 +697,14 @@ export function buildFieldsInitNodes(
       case isStatic && isPrivate && isMethod && !loose:
         needsClassRef = true;
         staticNodes.push(
+          buildPrivateStaticFieldInitSpec(prop, privateNamesMap),
+        );
+        staticNodes.unshift(
```

Comment:
Why unshift?

Reply:
I use `unshift` so that the declaration of the getter and/or setter occurs before the usage of the accessor(s) (e.g., when they're defined as the values on an object). Here's what a test failure looks like if that's changed to `push`:

```
    babel/packages/babel-plugin-proposal-private-methods/test/fixtures/static-accessors/basic/exec.js: expect(received).toEqual(expected) // deep equality

    Expected: "top secret string"
    Received: undefined
       1 | class Cl {
       2 |   static getValue() {
       3 |     return babelHelpers.classStaticPrivateFieldSpecGet(Cl, Cl, _privateStaticFieldValue);
       4 |   }
       5 |
       6 |   static setValue() {
       7 |     babelHelpers.classStaticPrivateFieldSpecSet(Cl, Cl, _privateStaticFieldValue, "dank");
       8 |   }
       9 |
      10 | }
      11 |
      12 | var _PRIVATE_STATIC_FIELD = {
      13 |   writable: true,
      14 |   value: "top secret string"
      15 | };
      16 | var _privateStaticFieldValue = {
      17 |   get: _get_privateStaticFieldValue,
      18 |   set: _set_privateStaticFieldValue
      19 | };
      20 |
      21 | var _get_privateStaticFieldValue = function () {
      22 |   return babelHelpers.classStaticPrivateFieldSpecGet(Cl, Cl, _PRIVATE_STATIC_FIELD);
      23 | };
      24 |
      25 | var _set_privateStaticFieldValue = function (newValue) {
      26 |   babelHelpers.classStaticPrivateFieldSpecSet(Cl, Cl, _PRIVATE_STATIC_FIELD, `Updated: ${newValue}`);
      27 | };
      28 |
      29 | expect(Cl.getValue()).toEqual("top secret string");
      30 | Cl.setValue();
      31 | expect(Cl.getValue()).toEqual("Updated: dank");
```

Reply:
Can we change the get/set variables to function declarations instead? If not, this is still fine.